routes/cards.js

```diff
@@ -1,8 +1,20 @@
 const router = require('express').Router();
+const { celebrate, Joi } = require('celebrate');
 const { getAllCards, createCard, deleteCard } = require('../controllers/cards');
 
 router.get('/cards', getAllCards);
-router.post('/cards', createCard);
-router.delete('/cards/:cardId', deleteCard);
+
+router.post('/cards', celebrate({
+  body: Joi.object().keys({
+    name: Joi.string().required().min(2),
+    link: Joi.string().required(),
```

Comment:
Нужно исправить: валидация поля `link` не соответствует схеме модели.
В модели `Card` поле `link` ожидается, что это будет не просто строка, а строка формата URL:
https://github.com/MaximMaslovVik/15_sprint/blob/5d5e9a2f50c14679a7121290a01a3d3631ea96cc/models/card.js#L3
Рекомендую использовать встроенный валидатор для указания схемы таких строк [`.uri()`](https://github.com/hapijs/joi/blob/master/API.md#stringurioptions):
```suggestion
    link: Joi.string().uri().required(),
```